#!/usr/bin/env python3
# _*_ coding:utf-8 _*_

PACKAGE = "rm_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("g_nms_thresh",double_t, 0, "g_nms_thresh", 0.1,0.1, 1.0)
gen.add("g_bbox_conf_thresh",double_t, 0, "g_bbox_conf_thresh", 0.1,0.1, 1.0)


exit(gen.generate(PACKAGE, "rm_detector", "dynamic"))
